src/restruct_poc/beams.hpp

```diff
@@ -0,0 +1,689 @@
+#pragma once
+
+#include <array>
+#include <numeric>
+
+#include <KokkosBlas.hpp>
+
+#include "beams_calcs.hpp"
+#include "interpolation.h"
+#include "types.hpp"
+
+#include "src/gebt_poc/quadrature.h"
+
+namespace oturb {
```

Comment:
If we are shortening the namespace name here from `openturbine`, let's go big -- what about `ot`?

Reply:
We can leave the namespace as `openturbine`. When I started this I had forgotten how namespaces work and thought that I'd have to prefix every type like `openturbine::View_LieAlgebraGroup`.

Reply:
Got it. It is kind of verbose though and I would have preferred a shorter name.

> When I started this I had forgotten how namespaces work and thought that I'd have to prefix every type like openturbine::View_LieAlgebraGroup.

Yup, that's fortunately not the case. That's why I pointed it out -- if all of our outermost code live under the same namespace (i.e. `openturbine`) we don't need to prefix it every time we call a type.